lib/i18n_sonder.rb

```diff
@@ -7,6 +7,7 @@
 require 'mobility/plugins/upload_for_translation'
 require 'i18n_sonder/workers/sync_approved_translations_worker'
 require 'i18n_sonder/workers/upload_source_strings_worker'
+require "i18n_sonder/railtie" if defined?(Rails)
```

Comment:
Is the condition needed?

Reply:
Technically, no, since the projects that currently include this gem are Rails projects. I guess there are other parts of this codebase that are Rails specific too, so maybe it's not necessary to be defensive like that here.